### PR TITLE
Improve swagger descriptions

### PR DESCRIPTION
**Pull request by okin**
This PR aims at improving the descriptions for the API.

### CODE REVIEW

**Review by Vad1mo — APPROVED** [review]

**Comment by okin** [comment]
@heww thank you for your review!

Is there another way to show that the endpoints of `/health` and `/ping` do not require any authentication?
The swagger documentation does not make this visible and I only found it out through [this comment](https://github.com/goharbor/harbor/issues/15624#issuecomment-924553645).

**Comment by Vad1mo** [comment]
@okin @heww can we have a look at this PR can merge it? There is not much missing to complete the PR

**Comment by Vad1mo** [comment]
> Can you remove the security in the getHealth and getPing APIs? These two definitions make a big change in the API handlers generated by the swagger.

According to the spec `security: []`means # No security which is correct in that case /health and /ping

**Comment by okin** [comment]
> @okin @heww can we have a look at this PR can merge it? There is not much missing to complete the PR

Sure, would love to see this getting merged. Right now I'm just a little bit unsure on how to progress this from my side. Is there anything I can do?

**Comment by OrlinVasilev** [comment]
Hi tehere @okin can you pleas check the comments! :) Thank you!

File: api/v2.0/swagger.yaml [hunk 1]
```diff
@@ -4258,8 +4259,9 @@ paths:
           $ref: '#/responses/500'
   /ping:
     get:
+      security: []
```
[the remark this answers is not in the document]
Comment (okin) [reply api/v2.0/swagger.yaml:4262]: Is there another way to show that this endpoint does not require any authentication?

The swagger documentation does not make this visible and I only found it out through https://github.com/goharbor/harbor/issues/15624#issuecomment-924553645.

File: api/v2.0/swagger.yaml [hunk 2]
```diff
@@ -21,6 +21,7 @@ security:
 paths:
   /health:
     get:
+      security: []
```
[the remark this answers is not in the document]
Comment (okin) [reply api/v2.0/swagger.yaml:24]: Is there another way to show that this endpoint does not require any authentication?

The swagger documentation does not make this visible and I only found it out through https://github.com/goharbor/harbor/issues/15624#issuecomment-924553645.

**Comment by okin** [comment]
> Hi tehere @okin can you pleas check the comments! :) Thank you!

Sure, I just added my questions regarding this directly to the comments.

**Comment by Vad1mo** [comment]
@okin, can you make changes on the bits we agree upon? The ones that don't touch `security` for the other changes you can make a separate PR.

This way we can accept your changes.

**Comment by okin** [comment]
> @okin, can you make changes on the bits we agree upon? The ones that don't touch `security` for the other changes you can make a separate PR.
> 
> This way we can accept your changes.

Sure. I've opened https://github.com/goharbor/harbor/pull/18205 for this.

**Pull request closed without merging**